tools/gyp/pylib/gyp/MSVSVersion.py

```diff
@@ -436,6 +448,7 @@ def _DetectVisualStudioVersions(versions_to_check, force_express):
       2015    - Visual Studio 2015 (14)
       2017    - Visual Studio 2017 (15)
       2019    - Visual Studio 2019 (16)
+      2022    - Visual Studio 2019 (17)
```

Comment:
```suggestion
      2022    - Visual Studio 2022 (17)
```

Reply:
Thanks, included in https://github.com/nodejs/gyp-next/pull/124